pynetbox/core/query.py

```diff
@@ -311,7 +311,7 @@ def req_all():
                     if first_run:
                         req = self._make_call(
                             add_params={
-                                "limit": req["count"],
+                                "limit": len(req["results"]),
```

Comment:
This is functionally doing the same thing. `len(req["results"])` and `req["count"]` would be the same number.